packages/specification/electronic-prescription-service-api.yaml

```diff
@@ -388,6 +388,11 @@ paths:
                 description: A valid acute, primary care prescription order request.
                 value:
                   $ref: examples/primary-care/acute/nominated-pharmacy/medical-prescriber/1-Process-Request-Send-200_OK.json
+              acute-multiple-dosages:
+                summary: Acute
```

Comment:
Please refrain from using identical names for different examples. 
Please indicate in the name that the example pertains to multiple dosage items.